hdfs_datanode/tox.ini

```diff
@@ -5,6 +5,8 @@ envlist =
     py{27,37}-{integration,unit}
```

Comment:
Hum, what about changing this line to `py{27,37}`, not sure why this is necessary `{integration,unit}`. That would fix the issue.